### ISSUE TITLE
Any missing prefixes that could be standardised?

### ISSUE THREAD

**Issue by matentzn** [biopragmatics/bioregistry#917]
The following list came out of this PR: https://github.com/mapping-commons/sssom-py/pull/397/files, as CURIES that cannot be standardised:

ABA
AEO_RETIRED
ANISEED
BAMS
BILS
BIRNLEX
BM
BSA
BilaDO
CALOHA
EHDAA2_RETIRED
EMAPA_RETIRED
FBbt_root
GAID
ILX
Image
KUPO
MAP
MESH
MIAA
NIFSTD_RETIRED
NIF_Subcellular
NLX
NLXANAT
OBO_REL
OGEM
OGES
OpenCyc
RETIRED_EHDAA2
TE
UBERONTEMP
Wikipedia
WikipediaCategory
XtroDO
ilxtr
ncithesaurus
nifext

Is this correct @cthoyt? Shouldn't we at least register some of the more widely used ones as prefix synonyms, like:

(numbers mean: occurrences in some ontology I happened to work on at the time)

Wikipedia: 477
CALOHA: 244 (the issue [here](https://github.com/biopragmatics/bioregistry/issues/721) seems to suggest this was, indeed, added)
MESH: 227
GAID: 191
OpenCyc: 182
MIAA: 93
BAMS: 73
BIRNLEX: 60

We can close this issue immediately, I just want to make sure the decision for not adding these and the reasoning behind it is documented again.

**Comment by cthoyt** [comment]
Yes, some of these are already registered as prefix synonyms. However, switching over to using `curies.Converter` in https://github.com/mapping-commons/sssom-py/pull/397 did not completely rewrite the loading to begin using synonyms. That will be another larger change, and I wanted to keep them separate. 

https://github.com/cthoyt/oquat is an automated tool that identifies _all_ prefixes across _all_ OBO Foundry ontologies that are not registered. I have put considerable effort into registering what I can, then trying to ask about ones that I could not figure out.  Two main issues I ran into is that people import lots of content from other ontologies that they don't understand and further most maintainers aren't responsive and able to answer questions (I have made many many issue requests for clarification)
